apps/organisations/forms.py

```diff
@@ -136,22 +134,9 @@ class OrganisationForm(forms.ModelForm):
         ),
         (
             "information",
-            RichTextUploadingFormField,
+            CKEditor5Widget,
```

Comment:
this should probabl be `CKEditor5Field`, not widget, sorry

Reply:
tried, didn't work. I try now with charfield and setting the ckeditor5widget as its widget

Reply:
ah, maybe it doesn't play nicely with parler because `CKEditor5Field` doesn't inherit from Charfield

Reply:
@goapunk seems working with charfield

Reply:
after staring at it for a while: I think ` OrganisationTranslation._meta.get_field( "information").formfield` should also work and we don't have to set the widget below then. Maybe worth testing it?

Reply:
Not sure where you want to test the formfield, instead of charfield? the issue with parler compatibility might come back. I leave it as charfield now. You can test it in your PR.